pkg/cfn/builder/vpc.go

```diff
@@ -38,6 +41,8 @@ const (
 	PrivateSubnetKey        = "PrivateSubnet"
 	PublicSubnetsOutputKey  = "SubnetsPublic"
 	PrivateSubnetsOutputKey = "SubnetsPrivate"
+
+	defaultDesiredMaskSize = 13
```

Comment:
Is this the result of `32-19=13` where `32 = defaultMaskSize` and `19 = defaultPrefix`? 
It might be worth adding `const defaultSubnetMask = 32` and `const defaultPrefix = 19`, then a func to calculate `defaultDesiredMaskSize`. It will make it easier to understand and maintain if either of these values change in the future.